src/rpc/blockchain.cpp

```diff
@@ -406,6 +409,24 @@ void entryToJSON(UniValue &info, const CTxMemPoolEntry &e)
     }
 
     info.pushKV("depends", depends);
+
+    std::set<std::string> setSpent;
+    for (unsigned int i = 0; i < tx.vout.size(); i++) {
+        auto it = mempool.mapNextTx.find(COutPoint(tx.GetHash(), i));
```

Comment:
Does this repeat tx.GetHash() for every vout?
Possibly makes sense to do it outside of the loop?
(Maybe the compiler is more intelligent than I'm imagining)

Reply:
It's fine as is, see `CTransaction::GetHash()`.